extensions/Webhooks/template/en/default/hook/global/user-error-errors.html.tmpl

```diff
@@ -0,0 +1,21 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% IF error == "select_an_event" %]
+  [% title = "Select at least one event" %]
+  You didn't select any event. Select at least one.
+
+[% ELSIF error == "define_a_name" %]
+  [% title = "Define a name" %]
+  You didn't define a name for your webhook. Define a name to identify your webhook.
+
+[% ELSIF error == "define_a_url" %]
+  [% title = "Define a URL" %]
+  You didn't define a URL for your webhook. Define one.
```

Comment:
Small request: Prefix all errror names with the extension name so avoid any naming conflicts.

webhooks_select_event
webhooks_define_name
webhooks_define_url